include/seqan3/alignment/pairwise/alignment_configurator.hpp

```diff
@@ -265,10 +266,14 @@ struct alignment_configurator
             using alignment_result_value_t = typename align_result_selector<std::remove_reference_t<wrapped_first_t>,
                                                                             std::remove_reference_t<wrapped_second_t>,
                                                                             config_t>::type;
-            using callback_on_result_t = std::function<void(alignment_result<alignment_result_value_t>)>;
+            using alignment_result_t = alignment_result<alignment_result_value_t>;
+            using callback_on_result_t = std::function<void(alignment_result_t)>;
             // Define the function wrapper type.
             using function_wrapper_t = std::function<void(indexed_sequence_pair_chunk_t, callback_on_result_t)>;
 
+            // Capture the alignment result type.
+            auto captured_cfg = cfg | align_cfg::alignment_result_capture<alignment_result_t>;
```

Comment:
I would suggest to rename this to `cfg_with_result_type`, (I saw this in the following PR)

because `align_cfg::alignment_result_capture` would be `align_cfg::result_type` with our current naming scheme

Reply:
I did rename the variable in #1760 and the config will be renamed somewhen later.